src/nvim/buffer_defs.h

```diff
@@ -1134,6 +1137,9 @@ struct window_S {
   pos_save_T w_save_cursor;         // backup of cursor pos and topline
   bool w_do_win_fix_cursor;         // if true cursor may be invalid
 
+  // Float preview window
+  int w_float_flags;                // FLOAT_ values
```

Comment:
I think we discussed this before, but for now this can just be single `bool`.  like `w_float_is_info` placed next to `w_floating` .

Reply:
yes. I forget there. because we don't have menu info confirm popupwindow. so use single `bool` is nice

Reply:
@bfredl  please review